phpBB/config/development/config.yml

```diff
@@ -17,3 +17,6 @@ core:
     extensions:
         composer_debug: true
         composer_verbose: true
+
+    finder:
+        not_use_cache: true
```

Comment:
and call it use_cache as suggested it is easier to read and understand :)

Reply:
Ah ok